package.json

```diff
@@ -76,6 +76,7 @@
     "@bbc/psammead-caption": "^2.2.28",
     "@bbc/psammead-consent-banner": "^2.3.30",
     "@bbc/psammead-copyright": "^1.2.26",
+    "@bbc/psammead-embed-error": "^1.0.0-alpha.19",
```

Comment:
Will this come out of alpha after a a11y swarm of this branch?

Reply:
Yes, @andrewscfc – hopefully tomorrow.